core-bundle/src/Controller/ContentElement/DownloadsController.php

```diff
@@ -83,7 +83,7 @@ protected function getFilesystemItems(Request $request, ContentModel $model): Fi
         };
 
         // Find filesystem items
-        $filesystemItems = FilesystemUtil::listContentsFromSerialized($this->filesStorage, $sources);
+        $filesystemItems = FilesystemUtil::listContentsFromSerialized($this->filesStorage, $sources ?? []);
```

Comment:
I think I would move this up to L82.

Reply:
Or should we allow `null` for `listContentsFromSerialized` instead? Serialized fields in Contao are always nullable.

Reply:
I'm fine with this as well. 👍